src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NSDbSinkWriter.scala

```diff
@@ -93,7 +88,7 @@ class NSDbSinkWriter(connection: NSDB,
     **/
   private def writeRecords(topic: String,
                            records: List[SinkRecord],
-                           kcqls: Array[ParsedKcql],
+                           kcqls: Array[MappingInterface],
```

Comment:
since we used a supertype as the array type, we should rename the parameter accordingly